cmd/up/up.go

```diff
@@ -79,7 +80,10 @@ func Up() *cobra.Command {
 			if err := upOptions.AddArgs(cmd, args); err != nil {
 				return err
 			}
-
+			if upOptions.DockerDesktop {
+				upOptions.Detach = true
+				os.Setenv(model.OktetoOriginEnvVar, model.OktetoDockerDesktopOrigin)
```

Comment:
Since if the only place where we need this information is [here](https://github.com/okteto/okteto/blob/c77c3537724bf769bba68585cf9f2ac1bae598b4/pkg/cmd/login/browser.go#L63) I think we should only have this line as a setter

Reply:
We also need it to know the origins while tracking where the commands come from